assets/js/components/legacy-notifications/setup-incomplete.js

```diff
@@ -41,8 +41,6 @@ export const getSetupIncompleteComponent = ( module, inGrid = false, fullWidth =
 	return ctaWrapper( cta, inGrid, fullWidth, createGrid );
 };
 
-export default getSetupIncompleteComponent;
```

Comment:
I know we're deleting this soon but I thought I'd fix it anyway :smile:

Reply:
Assuming this isn't used or only as a named import?

Reply:
Exactly that @aaemnnosttv . The two named exports are only imported once

![only-used-once](https://user-images.githubusercontent.com/8496063/126673569-81b4c71b-e482-4f0a-b081-3148ad7548ec.png)

`import { getSetupIncompleteComponent, getModuleInactiveComponent } from '../legacy-notifications/setup-incomplete';`